### ISSUE TITLE
Needs transpiling, Fails on IE11

### ISSUE THREAD

**Issue by ericmasiello** [mightyiam/get-own-enumerable-property-symbols#5]
Using this package in IE11 will fail. Could you add a build step to build down to ES5 code?

**Comment by mightyiam** [comment]
Sure. I'll do that.

**Comment by ericmasiello** [comment]
Might be related but I'm seeing this issue when I attempt to use Webpack and uglify my bundle that relies on this

```
ERROR in stencil.js from UglifyJs
SyntaxError: Unexpected token: operator (>) [./~/get-own-enumerable-property-symbols/index.js:1,0][stencil.js:43216,25]
```

**Comment by mightyiam** [comment]
Here is #6 for you, @ericmasiello.

**Comment by mightyiam** [comment]
Published v2.0.0.
